modules/ROOT/pages/apikit-odatav4-transform-to-sql-operation.adoc

```diff
@@ -0,0 +1,152 @@
+= Transforming to SQL Query
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can transform the input OData `GET` request to a SQL SELECT query with the `Transform to SQL Select` operation. This operation is useful when combined with the `select` operation of the Database Connector. 
+
+image::transform-to-sql-transformer.png["Transform to SQL select element and tab highlighted"]
+
+The output payload is a prepared statement and the attributes are a map of prepared statement parameters. 
+
+image::add-db-transform-to-sql.png["Select element and tab highlighted, as well as the attributes.queryparameters in the input parameters field"]
```

Comment:
```suggestion
image::add-db-transform-to-sql.png["Select the highlighted element and tab and the attributes.queryparameters in the input parameters field."]
```

Reply:
So sorry to say I don't understand your modification. Please, can you elaborate?